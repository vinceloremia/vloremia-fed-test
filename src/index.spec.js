import React from 'react';

jest.mock(
	'./App',
	() =>
		function App() {
			return <div>Mock App</div>;
		}
);

describe('App Mount', () => {
	it('mounts the App into the DOM root node', () => {
		const rootElement = document.createElement('div');
		rootElement.id = 'root';
		document.body.appendChild(rootElement);

		require('./index.js'); // eslint-disable-line global-require

		expect(rootElement.innerHTML).toMatchSnapshot();
	});
});
