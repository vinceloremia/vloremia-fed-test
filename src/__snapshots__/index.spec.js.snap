// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Mount mounts the App into the DOM root node 1`] = `"<div>Mock App</div>"`;
